### PR TITLE
#141409243 Setup unit testing for the front end

### PR DESCRIPTION
**Pull request by mercyoseni**
#### What does this PR do?
- It enables unit testing for the front-end and allows execution of tests locally during development.
#### Description of Task to be completed?
- To ensure the front end has all functionality work the way it was intended. Use Jasmine and Karma (with PhantomJS/Chrome/Firefox) to setup front end testing environment.
#### How should this be manually tested?
- run `npm install`
- run `karma start`
#### Any background context you want to provide?
- Initially, there was no way to test the front-end, hence the need to setup unit testing for the front-end.
#### What are the relevant pivotal tracker stories?
- NA
#### Screenshots (if appropriate)
screen shot 2017-03-15 at 4 46 31 pm
screen shot 2017-03-15 at 4 49 02 pm

#### Questions:
- NA

### CODE REVIEW

File: test/sample-spec.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe("Sample test",function(){
```
Comment (q-ode) [inline test/sample-spec.js]: Please check through the style guides and ensure this conforms.

Comment (mercyoseni) [reply test/sample-spec.js]: Does this conform?
describe("Sample test", function() {
  it('This is a sample test', function() {
    expect(true).toEqual(true);
  });
});

Comment (q-ode) [reply test/sample-spec.js]: Not yet. ES6 Arrow functions nko?

Comment (mercyoseni) [reply test/sample-spec.js]: Alright.

Comment (mercyoseni) [reply test/sample-spec.js]: Done.

File: test/sample-spec.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('Sample test', () => {
+  it('This is a sample test', () => {
+    expect(true).toEqual(true);
```
[the remark this answers is not in the document]
Comment (uopaluwa) [reply test/sample-spec.js:3]: expect is defined in testing framework

**Review by uopaluwa — APPROVED** [review]
Check out

File: karma.conf.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+// Karma configuration
+// Generated on Wed Mar 15 2017 13:39:20 GMT+0100 (WAT)
+
+module.exports = function (config) {
```
[the remark this answers is not in the document]
Comment (uopaluwa) [reply karma.conf.js:4]: not necessary for exports

[the remark this answers is not in the document]
Comment (uopaluwa) [reply karma.conf.js:4]: not necessary for exports

**Pull request merged**